darwin/dataset/remote_dataset.py

```diff
@@ -249,7 +250,7 @@ def pull(
 
         if release.latest:
             latest_dir = self.local_releases_path / "latest"
-            if latest_dir.exists():
+            if latest_dir.is_symlink():
```

Comment:
This change seems to be working on macOS, but I don't have access to a Windows machine. Do you happen to have further information on why it breaks on Windows? That would be super helpful 🙂 

Also, is there a reason why we didn't like `latest_dir.exists()`?

Reply:
`latest_dir.exists()` breaks on my Windows machine, `is_symlink` and the os alternative work.
I think `exists()` is meant for directories and files only, Windows handles symlinks a bit different as Unix systems, wherefore it does not work.

Reply:
Maybe you can add some unit-tests and schedule them on different machines via GitHub actions.